### PR TITLE
Draft: Support Dockerized Workflows

### PR DESCRIPTION
**Pull request by chriswininger**
This pr attempts to add a flow for building docker images of our app.

I attempt to create easy scripts for both building and running individual services as well as collectively running all services

This PR is still in a draft phase and not everything is working yet.

TODO:

* Fix places that refer to APP_URL
* Create scripts for tagging and publishing staging/prod images

### CODE REVIEW

File: web-portal/layouts/default.vue [hunk 1]
```diff
@@ -9,6 +9,7 @@
     </ii-toolbar>
     <main>
       <!-- Content -->
+      <div> {{ $config.API_URL}} </div>
```
Comment (MatthewGidcomb) [inline web-portal/layouts/default.vue]: Cleanup nitpick

File: web-portal/plugins/vue-moment.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import Vue from 'vue'
 import VueMoment from 'vue-moment'
 
+console.log(Vue.$config)
```
Comment (MatthewGidcomb) [inline web-portal/plugins/vue-moment.js]: Cleanup nitpick

File: web-portal/layouts/default.vue (hunk 1 above)
Comment (chriswininger) [reply web-portal/layouts/default.vue]: whoops lol

File: web-portal/plugins/vue-moment.js (hunk 2 above)
Comment (chriswininger) [reply web-portal/plugins/vue-moment.js]: thx

**Pull request merged**